lib/std/compress/zstandard.zig

```diff
@@ -295,24 +286,6 @@ test "zero sized block" {
 
     try expectEqualDecoded("", input_raw);
     try expectEqualDecoded("", input_rle);
-}
-
-test "zero sized block streaming" {
-    // default stack size for wasm32 is too low for Decompressor - slightly
-    // over 1MiB stack space is needed via the --stack CLI flag
-    if (@import("builtin").target.cpu.arch == .wasm32) return error.SkipZigTest;
-
-    const input_raw =
-        "\x28\xb5\x2f\xfd" ++ // zstandard frame magic number
-        "\x20\x00" ++ // frame header: only single_segment_flag set, frame_content_size zero
-        "\x01\x00\x00"; // block header with: last_block set, block_type raw, block_size zero
-
-    const input_rle =
-        "\x28\xb5\x2f\xfd" ++ // zstandard frame magic number
-        "\x20\x00" ++ // frame header: only single_segment_flag set, frame_content_size zero
-        "\x03\x00\x00" ++ // block header with: last_block set, block_type rle, block_size zero
-        "\xaa"; // block_content
-
```

Comment:
looks like you deleted a lot more than just the `if (@import("builtin").target.cpu.arch == .wasm32) return error.SkipZigTest;` check?

Reply:
I see, you merged the tests into one.

Reply:
Yeah - the tests were split up when wasm32-wasi needed more stack space so I reverted that as well. They could remain split-up by decompression API used, but I don't think it makes much difference and this way there is a bit less test code to read through or get out of sync.